nilearn/glm/first_level/first_level.py

```diff
@@ -864,6 +891,17 @@ def first_level_from_bids(dataset_path,
         Filter examples would be ('desc', 'preproc'), ('dir', 'pa')
         and ('run', '10').
 
+    slice_time_ref : :obj:`float` between 0.0 and 1.0, optional
+        This parameter indicates the time of the reference slice used in the
+        slice timing preprocessing step of the experimental runs. It is
+        expressed as a percentage of the t_r (time repetition), so it can have
```

Comment:
Actually, it's not a percentage  it's a fraction

Reply:
indeed

Reply:
changed it also in the FirstLevelModel doc string